analyses/molecular_classifying_EWS/01-reclassify_as_ewings.Rmd

```diff
@@ -0,0 +1,64 @@
+---
+title: "Reclassify integrated_diagnosis,short_histology metadata column for samples that have hallmark Ewings Sarcoma fusions"
+output: html_notebook
+author: K S Gaonkar for D3b
+date: January 2020
+---
+
+Identify "integrated_diagnosis","short_histology" from clinical file for sample IDs with hallmark EWSR1 fusions and reclassify as Ewings Sarcoma (integrated_diagnosis) and EWS (short_histology)
+
+```{r}
+library("tidyverse")
+```
+
+
+### Read in data
+
+```{r}
+# to get root directory
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+# data directory
+dataDir <- file.path(root_dir, "data")
+# histology files
+clinical <- readr::read_tsv(file.path(dataDir, "pbta-histologies.tsv")) %>%
+  select("Kids_First_Biospecimen_ID","integrated_diagnosis","short_histology","broad_histology","Kids_First_Participant_ID","experimental_strategy")
+# get ewings fusion calls from fusion-summary results file
+ewings_foi_bsids <- readr::read_tsv(file.path(root_dir,"analyses","fusion-summary","results","fusion_summary_ewings_foi.tsv")) 
+
+# get subset folder
+results_dir<-file.path(root_dir,"analyses","molecular_classifying_EWS","results")
+
+# create if doesn't exist
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+
+
+```
+
+### reclassify integrated diagnosis and short_histology of samples that have hallmark ewings sarcoma EWSR1 fusion
+```{r}
+reclassifySamplesEwings<-ewings_foi_bsids %>% 
+  dplyr::mutate(integrated_diagnosis_reclassified=dplyr::case_when(.data$`EWSR1--FLI1` !=0 | .data$`EWSR1--PATZ1` !=0 | .data$`EWSR1--ERG` !=0  |.data$`FUS--ERG` !=0  ~ "Ewings Sarcoma"),
+                short_histology_reclassified=dplyr::case_when(.data$`EWSR1--FLI1` !=0 | .data$`EWSR1--PATZ1` !=0 | .data$`EWSR1--ERG` !=0  |.data$`FUS--ERG` !=0 ~ "EWS"))
+
+clinical<-clinical %>% left_join(reclassifySamplesEwings ,by=c("Kids_First_Biospecimen_ID"))
+
+readr::write_tsv(clinical,file.path(results_dir,"reclassified_ewings_samples.tsv"))
+
+
+```
+
+### list which samples are reclassfied
+```{r}
+# print RNAseq samples with ewings sarcoma hallmark fusions 
+check<-clinical %>% dplyr::filter(!is.na(integrated_diagnosis_reclassified) & !is.na(short_histology_reclassified))
+
+check
+
+# check what other datatype is associated with this participants which belong to the newly reclassified ewings sarcoma samples
+clinical %>% dplyr::filter(Kids_First_Participant_ID %in% check$Kids_First_Participant_ID) %>% select(experimental_strategy,Kids_First_Biospecimen_ID,Kids_First_Participant_ID,integrated_diagnosis,integrated_diagnosis_reclassified) %>% dplyr::arrange(Kids_First_Participant_ID)
```

Comment:
Can you add a new line after each `%>%` here please? I think it would make this more readable.